api/v2/api.md

```diff
@@ -0,0 +1,344 @@
+# dcrtime API Specification
+
+## V2
+
+This document describes the REST API provided by a `dcrtimed` server. This API allows users to create and upload hashes which are periodically submitted to the Decred blockchain. It gives the option to send a single string digest, as well as multiples in a array of string digests. It also provides the ability to confirm the addition of the hash to a timestamped collection along with showing and validating their inclusion in the Decred blockchain.
```

Comment:
Keep lines to a 80 column max.

Apply throughout document.

Reply:
Wasn't sure if it also applied to the docos. Updated :+1: